local/run.go

```diff
@@ -406,11 +408,25 @@ func (a *Agent) Run(ctx context.Context) error {
 		`BUILDKITE_PLUGINS_PATH=`+pluginsDir,
 	)
 
+	// Windows requires certain env variables to be present
+	if runtime.GOOS == "windows" {
+		cmd.Env = append(cmd.Env,
+			"PATH="+os.Getenv("PATH"),
+			"SystemRoot="+os.Getenv("SystemRoot"),
+			"WINDIR="+os.Getenv("WINDIR"),
+			"COMSPEC="+os.Getenv("COMSPEC"),
+			"PATHEXT="+os.Getenv("PATHEXT"),
+			"TMP="+os.Getenv("TMP"),
+			"TEMP="+os.Getenv("TEMP"),
+			"SYSTEMDRIVE="+os.Getenv("SYSTEMDRIVE"),
+		)
+	}
+
 	// this function is called at the end of Run()
 	// it kills the agent
 	a.stopFunc = func() error {
 		defer os.Remove(bootstrap.Name())
-		_ = cmd.Process.Signal(os.Interrupt)
+		_ = cmd.Process.Kill()
```

Comment:
Signals do not really work on windows, I have ran into this issue many times in the past.

Reply:
Worth wrapping inside a runtime.GOOS=="windows" conditional so the other OS' are not brutalised for Windows' failing?

Reply:
Good idea! Will do.